db/institutions.yml

```diff
@@ -680,6 +680,39 @@ enthought:
   updated: 2020-12-29 04:16:46.030159
   uuid: 358f4e2b-e0dc-44e6-8cf4-605409118494
   zip: '78701'
+epfl:
+  aka:
+    - EPFL,
+    - EPFL
+    - Valais,
+    - EPFL
+    - Wallis
+  city: Sion
+  country: Switzerland
+  date: 2023-09-14
+  departments:
+    isic:
+      name: Institute of Chemical Sciences and Engineering
+      aka:
+        - Institut
```

Comment:
this may need a quick fix.  the entries in the gui are space separated so with the gui you would have had to do "Institute des sciences....." with the quotes for it to work.  To fix it, just edit the file directly.